entrypoint.sh

```diff
@@ -6,24 +6,42 @@ response=$(curl \
         -H "Content-Type: application/json" \
         -s "https://api.cloudflare.com/client/v4/zones/$CLOUDFLARE_ZONE_ID/dns_records?name=$RECORD_NAME.$RECORD_DOMAIN")
 
-recordIds=$(jq -r  '.result[].id' <<< "${response}" ) 
+success=$(jq -r  '.success' <<< "${response}" )
 
-if [ ${#recordIds[@]} -eq 0 ]; then
-    echo "Pages Update: No record found. Follow Cloudflare setup instructions to add an initial IPFS record for $RECORD_NAME"
-else
-  echo "Pages Update: DNS Record Found"
-  for recordId in "${recordIds[@]}"
-  do
+if [ $success != "true" ]; then
+  echo "Pages Update: Failed to update record!"
+  errors=$(jq -r  '.errors' <<< "${response}" )
+  echo "Errors: $errors"
+  exit 1
+fi
+
+totalCount=$(jq -r  '.result_info.total_count' <<< "${response}" ) 
+if [ $totalCount == "0" ]; then
+  echo "Pages Update: No record found. Follow Cloudflare setup instructions to add an initial IPFS record for '$RECORD_NAME'"
+  exit 1
+fi
+
+echo "Pages Update: DNS Record Found!"
+
+recordIds=$(jq -r  '.result[].id' <<< "${response}" )
+for recordId in "${recordIds[@]}"
```

Comment:
If I am not mistaking we should expect this to always only include one id at most since there will only be one dnslink TXT record for `RECORD_DOMAIN`, so maybe we can just get the first items instead of doing a loop for clarity?

Reply:
sure! feel free to submit another PR if you get around to it. I don't mind taking this one as is. i noticed there is also an issue with PR tests, I can look into fixing that later. as long as the test succeeds on merge we should be okay.